examples/websocket-client-data-binding/websocket_client_data_binding.bal

```diff
@@ -0,0 +1,21 @@
+import ballerina/log;
+import ballerina/websocket;
+
+type Person record {|
+    string name;
+    int age;
+|};
+
+public function main() returns error? {
+   // Create a new [WebSocket client](https://lib.ballerina.io/ballerina/websocket/latest/clients/Client).
+   websocket:Client wsClient = check new("ws://localhost:9090/basic/ws");
+
+   Person personRecord = {name: "John", age: 25};
+
+   // Write the created Person record to the server.
+   check wsClient->writeMessage(personRecord);
+
+   // Read the message echoed from the server and bind it to the Person record.
+   Person person = check wsClient->readMessage();
+   log:printInfo("Person name: " + person.name);
```

Comment:
Shall we use string template here rather than string concatenation ? Or else we could do like following:
```ballerina
log:printInfo("Received response: ", personName = person.name);
```